.github/workflows/go-pr.yml

```diff
@@ -1,8 +1,9 @@
 name: Go (PR)
 
 on:
+  branches: [ "v2" ]
+  
```

Comment:
This Workflow must run only on pull requests of v2 branch. Please rollback this line